product_docs/docs/biganimal/release/pricing_and_billing/index.mdx

```diff
@@ -74,8 +76,27 @@ Enabling [Apache Superset](/biganimal/latest/using_cluster/06_analyze_with_super
 
 Enabling [PgBouncer](../getting_started/creating_a_cluster/#pgbouncer) to pool your connections incurs additional costs that depend on your cloud provider. In addition to the cloud provider costs, PgBouncer connects to your primary server and requires an IP address. BigAnimal provisions up to three instances per PgBouncer-enabled cluster to ensure that performance is unaffected, so each availability zone receives its own instance of PgBouncer. The extra VM costs are the 2vcpu SKU times the number of PgBouncer instances. For AWS, the instance type is c5.large. For Azure, the instance type is F2s_v2.
 
-## Billing
+## Payments and billing
+
+On the [Billing](https://portal.biganimal.com/billing) page, you can manage your payment methods and view your invoice total.
```

Comment:
the [Billing](https://portal.biganimal.com/billing) page is only visible for `Digital self-service using a credit card` service user. 

It is not visible to the Azure Marketplace and SOF customers.